src/main/java/com/firebolt/jdbc/client/query/StatementClientImpl.java

```diff
@@ -62,7 +62,9 @@ public class StatementClientImpl extends FireboltClient implements StatementClie
 	//private static final String QUERY_ID_FETCHER = "select query_id from information_schema.query_history where status = 'STARTED_EXECUTION' and  query_label = ?";
 	private static final String QUERY_ID_FETCHER = "select query_id from information_schema.query_history where status = 'STARTED_EXECUTION' and  query_text like ?";
 	private final BiPredicate<Call, String> isCallWithLabel = (call, label) -> call.request().tag() instanceof String && Objects.equals(call.request().tag(), label);
-	static final String HEADER_UPDATE_PARAMETER = "Firebolt-Update-Parameters";
+	public static final String HEADER_UPDATE_PARAMETER = "Firebolt-Update-Parameters";
+	public static final String HEADER_UPDATE_ENDPOINT = "Firebolt-Update-Endpoint";
+	public static final String HEADER_RESET_SESSION = "Firebolt-Update-Session";
```

Comment:
typo, `Firebolt-Reset-Session` header according to the spec

Reply:
fixed